coala_quickstart/generation/Settings.py

```diff
@@ -52,6 +52,8 @@ def generate_ignore_field(project_dir,
         A comma-separated string containing the globs to ignore.
     """
 
+    ignore_globs = list(ignore_globs)
+
```

Comment:
I don't think this is required because you've already converted it to a list [here](https://github.com/coala/coala-quickstart/pull/209/files#diff-7e6eaad4d79d2cfe0c15b00a0c204498R53) which is returned [here](https://github.com/coala/coala-quickstart/blob/master/coala_quickstart/coala_quickstart.py#L81) and further passed on to `generate_settings` function and finally ends up here.